### PR TITLE
Add AutomobileInsurance protocol

### PR DESCRIPTION
**Pull request by randunel**

### CODE REVIEW

File: src/schema/automobile-insurance.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            "experimental": true,
+            "description": "A list of available cover options found on the page.",
+            "minItems": 1,
+            "maxItems": 9,
+            "items": { "$ref": "#/domains/AutomobileInsurance/types/Cover" }
+        },
+        "Cover": {
+            "type": "object",
+            "experimental": true,
+            "description": "Flight fare information.",
```
Comment (marcusgreenwood) [inline src/schema/automobile-insurance.json]: ummm

File: src/schema/index.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ module.exports = {
         Generic: require('./generic'),
         FlightBooking: require('./flight-booking'),
         VacationRental: require('./vacation-rental'),
+        AutomobileInsurance: require('./automobile-insurance'),
```
Comment (marcusgreenwood) [inline src/schema/index.js]: I know Automobile sounds more sophisticated, but I think that rules out motorbikes whereas we'd probably include them under this banner. I'd rename to `MotorInsurance`

File: src/schema/automobile-insurance.json [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+            "additionalProperties": false,
+            "properties": {
+                "name": {
```
Comment (marcusgreenwood) [inline src/schema/automobile-insurance.json]: How does one "skip" - i.e. select none of the options? I had it as an extra option as `skip:true`

File: src/schema/automobile-insurance.json [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    "experimental": true,
+    "inputs": {
+        "url": {
+            "typeRef": "#/domains/Generic/types/URL",
+            "description": "Website entry point.",
+            "initial": true
+        },
+        "account": {
+            "typeRef": "#/domains/Generic/types/Account",
+            "description": "Account information for filling in contact details.<br/>Receipts and booking references will typically be sent to specified <code>email</code>.<br/>Some websites also require registering user account, in which case <code>password</code> must be provided.",
```
Comment (inca) [inline src/schema/automobile-insurance.json]: I see same description across all 3 domains, so just remove it from here and move it to `generic.json` instead.

File: src/schema/index.js (hunk 2 above)
Comment (randunel) [reply src/schema/index.js]: I insure my motorcycle and the flow is different. The post-deeplink might be the same.

**Pull request merged**